### PR TITLE
Update Mentor July Ad-Hoc Availability

### PR DESCRIPTION
**Pull request by khairahscorner**
## Description
- Added more mentors and their ad-hoc availabilities
- Sorted WCC directors and leads with July availability for more visibility

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [x] Mentor Update
- [ ] Data Update
- [ ] Documentation
- [ ] Other

## Screenshots
Screenshot 2024-07-01 at 7 05 11 pm

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -231,7 +231,7 @@
 - name: Sonali Goel
   disabled: false
   matched: false
-  sort: 200
+  sort: 300
```
Comment (dricazenck) [inline _data/mentors.yml]: I don't think Sonali will be more available this month. So please check with her first.

File: _data/mentors.yml [hunk 2]
```diff
@@ -668,7 +668,7 @@
     I am Frontend engineer with 4years of experience. I have had opportunities of working with several companies and building sustainable products for them. I am very keen to solving problems and helping people as well
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
    I am Frontend engineer with 4 years of experience. I have had opportunities of working with several companies and building sustainable products for them. I am very keen to solving problems and helping people as well
```

**Review by dricazenck — APPROVED** [review]

File: _data/mentors.yml (hunk 1 above)
Comment (khairahscorner) [reply _data/mentors.yml]: Oh okay, it's because she recently filled the form to update her availability for July to November

Comment (dricazenck) [reply _data/mentors.yml]: yes, unfortunately some personal issue came this week.

Comment (khairahscorner) [reply _data/mentors.yml]: ☹️ I've removed her availability for July

**Pull request merged**